main/src/main/java/com/sedmelluq/discord/lavaplayer/source/soundcloud/DefaultSoundCloudPlaylistLoader.java

```diff
@@ -95,13 +95,19 @@ protected AudioPlaylist loadFromSet(
         .map(dataReader::readTrackId)
         .collect(Collectors.toList());
 
-    List<JsonBrowser> trackDataList;
+    int numTrackIds = trackIds.size();
+    List<JsonBrowser> trackDataList = new ArrayList<>();
 
-    try (CloseableHttpResponse response = httpInterface.execute(new HttpGet(buildTrackListUrl(trackIds)))) {
-      HttpClientTools.assertSuccessWithContent(response, "track list response");
+    for (int i = 0; i < numTrackIds; i += 50) {
+      int last = Math.min(i + 49, numTrackIds);
```

Comment:
Since end parameter of `subList` is exclusive, shouldn't this be `+ 50`?

Reply:
Ah, yeah